.github/workflows/release-drafter.yml

```diff
@@ -1,3 +1,5 @@
+# Generates the draft release for the js-sdk
+# XXX: When would we run this via a dispatch?
```

Comment:
When element web builds, so element desktop has up to date release notes

Reply:
The js-sdk one? Do tghey propagate all the way through the chain again?

Reply:
Oh sorry thought this was the ED repo. The workflow_dispatch is for manual triggering, in case of failure/workflow updates etc. The repository_dispatch on ED's is for EW triggering it

Reply:
ah, gotcha. Hopefully this is accurate then?